File: URI-Java/1041.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Locale.setDefault(Locale.US);
        Scanner entrada = new Scanner(System.in);

        double x = entrada.nextDouble();
        double y = entrada.nextDouble();

        if(x == 0 && y == 0)
            System.out.println("Origem");
        else if(y == 0 && x != 0)
            System.out.println("Eixo X");
        else if(x == 0 && y != 0)
            System.out.println("Eixo Y");
        else if(x > 0 && y > 0)
            System.out.println("Q1");
        else if(x < 0 && y > 0)
            System.out.println("Q2");
        else if(x < 0 && y < 0)
            System.out.println("Q3");
        else if(x > 0 && y < 0)
            System.out.println("Q4");
 
    }
 
}

File: C++/1041.cpp
#include <iostream>

using namespace std;

string coordenadas (double x, double y){
  
    if(!x&&!y)
        return "Origem";
    if(x==0)
        return "Eixo Y";
    if (y==0)
        return "Eixo X";
    if (x>0 && y>0)
        return  "Q1";
    if (x<0 && y>0)
        return  "Q2";
     if (x<0 && y<0)
        return  "Q3";

    return  "Q4";

}

int main(void) {
 
    double x,y;
    cin >> x >> y;
  
    cout << coordenadas(x, y) << endl;
    return 0;
}